.gitlab-ci.yml

```diff
@@ -484,6 +524,19 @@ deploy_doxygen_documentation:
     - cd ${CI_PROJECT_DIR}/build/doc/doxygen/html &&
       rsync -avz --delete -e "ssh -i ${HOME}/.ssh/espresso_rsa" ./ espresso@elk.icp.uni-stuttgart.de:/home/espresso/public_html/html/dox
 
+deploy_tutorials:
+  extends: .deploy_base
+  dependencies:
+    - run_tutorials
+  script:
+    - cd ${CI_PROJECT_DIR}/build/doc/tutorials
+    - 'grep -Poe " src=(.)figures/.+?\1" */*.html > deploy_list.txt'
```

Comment:
what is happening here? The usage of regex should be documented somehow...

Reply:
Yes, that's quite inelegant. I'll write a proper Python script without regex for this job.

Reply:
Fixed by 638377d.